fluent-langneg/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
+    "target": "es2020",
```

Comment:
Is this enough? I was under the impression that TS only added the `Intl.Locale` types in `es2021`.